docs/khaas/getting-started.md

```diff
@@ -0,0 +1,57 @@
+# KubeHound as a Service (KHaaS)
+
+KHaaS enables you to use KubeHound in a distributive way. It is being splitted in 2 main categories:
+
+- The ingestor stack which includes the `graphdb`, `storedb`, `UI` and `grpc endpoint`.
+- The collector (the kubehound binary) which will dump and send the k8s resources to the KHaaS `grpc endpoint`.
+
+[![](../images/khaas-architecture.png)](../images/khaas-architecture.png)
+
+!!! note
+
+    You need to deploy the data storage you want to use ([AWS s3 in our example](https://github.com/DataDog/KubeHound/tree/main/deployments/terraform)).
+
+## Automatic collection
+
+KHaaS has been created to deployed inside a kubernetes cluster. This way you can set through Kubernetes `CronJob` daily dumps of your infrastucture. To configure and deploy it, please refer to the [deployment](deployment.md) section.
+
+## Manual collection
+
+In order to use `kubehound` with KHaaS, you need to specify the api endpoint you want to use. Since this is not likely to change in your environment, we advise you to use the local config file. By default KubeHound will look for `./kubehound.yaml` or `$HOME/.config/kubehound.yaml`. As example here we set the default endpoint with disabled SSL.
```

Comment:
```suggestion
In order to use `kubehound` with KHaaS, you need to specify the api endpoint you want to use. Since this is not likely to change in your environment, we recommend using the local config file. By default KubeHound will look for `./kubehound.yaml` or `$HOME/.config/kubehound.yaml`. For instance if the default configuration we set the endpoint with disabled SSL:
```